collectors/nfacct.plugin/plugin_nfacct.c

```diff
@@ -879,9 +881,11 @@ int main(int argc, char **argv) {
 
         fflush(stdout);
 
-        // restart check (14400 seconds)
-        if(now_monotonic_sec() - started_t > 14400) break;
+        if (now_monotonic_sec() - started_t > NFACCT_RESTART_EVERY_SECONDS) {
+            collector_info("NFACCT reached my lifetime expectancy. Exiting to restart.");
```

Comment:
This message looks a bit strange to me to be honest, but I copy/paste it from freeipmi

https://github.com/netdata/netdata/blob/23118ba3a695729f9eb2b889834dde2c0f9e3be3/collectors/freeipmi.plugin/freeipmi_plugin.c#L1894-L1895